Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Action.swift

```diff
@@ -76,4 +96,96 @@ public class Action {
+                    return Disposables.create {}
+                }
+            } catch {
+                completable(.error(error))
+                return Disposables.create {}
+            }
+        }
+    }
+    
+    public func returnToLaunch() -> Completable {
+        return Completable.create { completable in
+            let rtlRequest = Dronecore_Rpc_Action_ReturnToLaunchRequest()
+            
+            do {
+                let rtlResponse = try self.service.returntolaunch(rtlRequest)
+                if (rtlResponse.actionResult.result == Dronecore_Rpc_Action_ActionResult.Result.success) {
+                    completable(.completed)
+                    return Disposables.create {}
+                } else {
+                    completable(.error("Cannot return to launch: \(rtlResponse.actionResult.result)"))
+                    return Disposables.create {}
+                }
+            } catch {
+                completable(.error(error))
+                return Disposables.create {}
+            }
+        }
+    }
+    
+    public func transitionToFixedWings() -> Completable {
+        return Completable.create { completable in
+            let toFixedWingsRequest = Dronecore_Rpc_Action_TransitionToFixedWingsRequest()
+            
+            do {
+                let toFixedWingsResponse = try self.service.transitiontofixedwings(toFixedWingsRequest)
+                if (toFixedWingsResponse.actionResult.result == Dronecore_Rpc_Action_ActionResult.Result.success) {
+                    completable(.completed)
+                    return Disposables.create {}
+                } else {
+                    completable(.error("Cannot transition to fixed wings: \(toFixedWingsResponse.actionResult.result)"))
+                    return Disposables.create {}
+                }
+            } catch {
+                completable(.error(error))
+                return Disposables.create {}
+            }
+        }
+    }
+    
+    public func transitionToMulticopter() -> Completable {
+        return Completable.create { completable in
+            let toMulticopterRequest = Dronecore_Rpc_Action_TransitionToMulticopterRequest()
+            
+            do {
+                let toMulticopterResponse = try self.service.transitiontomulticopter(toMulticopterRequest)
+                if (toMulticopterResponse.actionResult.result == Dronecore_Rpc_Action_ActionResult.Result.success) {
+                    completable(.completed)
+                    return Disposables.create {}
+                } else {
+                    completable(.error("Cannot transition to multicopter: \(toMulticopterResponse.actionResult.result)"))
+                    return Disposables.create {}
+                }
+            } catch {
+                completable(.error(error))
+                return Disposables.create {}
+            }
+        }
+    }
+    
+    
+    public func getTakeoffAltitude() -> Float {
```

Comment:
Can we make a `Single` out of that? So that it is still an RX stream and it can be combined with the others in the reactive way :-).